### PR TITLE
Hotfix/fix addis ababa

### PR DESCRIPTION
**Pull request by kms6bn**
updated PR to be against develop

### CODE REVIEW

**Comment by kms6bn** [comment]
Hey @erinspace! Let me know if I fixed all your concerns.

File: scrapi/harvesters/addis_ababa.py [hunk 1]
```diff
@@ -6,6 +6,27 @@
 from __future__ import unicode_literals
 
 from scrapi.base import OAIHarvester
+from scrapi.base import helpers
+
+
+def format_uris_addis_ababa(*args):
+    identifiers = helpers.gather_identifiers(args)
+    provider_uris, object_uris = helpers.seperate_provider_object_uris(identifiers)
```
Comment (fabianvf) [inline scrapi/harvesters/addis_ababa.py]: Is there ever more than one `provider_uri`? If there is, you are overwriting this variable below, which may result in information loss.

File: scrapi/harvesters/addis_ababa.py [hunk 2]
```diff
@@ -6,6 +6,27 @@
...
+
+    for arg in args:
+        if arg and 'http://hdl.handle.net/123456789/' in arg[0]:
+            doc_id = arg[0].replace('http://hdl.handle.net/123456789/', '')
+            canonical_uri = 'http://etd.aau.edu.et/handle/123456789/' + doc_id
+            provider_uris = ['http://etd.aau.edu.et/handle/123456789/' + doc_id]
```
Comment (fabianvf) [inline scrapi/harvesters/addis_ababa.py]: Here is where you may be turning `provider_uris` from a list with many elements to a list with one element. Also, it doesn't look like you are using the `provider_uris` that you make above, does it not pull out the right information?

File: scrapi/harvesters/addis_ababa.py [hunk 3]
```diff
@@ -6,6 +6,27 @@
...
+
+
+def format_uris_addis_ababa(*args):
+    identifiers = helpers.gather_identifiers(args)
+    provider_uris, object_uris = helpers.seperate_provider_object_uris(identifiers)
+
+    for arg in args:
+        if arg and 'http://hdl.handle.net/123456789/' in arg[0]:
+            doc_id = arg[0].replace('http://hdl.handle.net/123456789/', '')
+            canonical_uri = 'http://etd.aau.edu.et/handle/123456789/' + doc_id
```
Comment (fabianvf) [inline scrapi/harvesters/addis_ababa.py]: This is sort of a minor quibble, but `str.format` is usually the preferred way to format strings: 
```python
canonical_uri = 'http://etd.aau.edu.et/handle/123456789/{}'.format(doc_id)
```

**Comment by fabianvf** [comment]
Why did you close this PR?

**Comment by fabianvf** [comment]
Oh, never mind. Missed the branch!

**Pull request closed without merging**